### PR TITLE
Correctly handle labels for new metrics objects

### PR DESCRIPTION
**Pull request by digitalsadhu**

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -252,6 +252,35 @@ test('metrics with labels', done => {
...
+    const consumer = new PrometheusMetricsConsumer({ client: promClient });
+
+    const source = src([
+        {
+            name: 'my_new_counter_with_labels',
+            description: '.',
+            podlet: 'recommendations',
+            url: 'http://mylayout.com',
+            method: 'GET',
+            status: 200,
```
Comment (trygve-lie) [inline test/index.test.js]: Any resons for the `podlet`, `url`, `method` and `status` fields? Would'nt it be better to test with an instance of the metric class here also?

Comment (digitalsadhu) [reply test/index.test.js]: yeah, just a quick fix copy and paste from another test so I can get this fixed up.

Comment (digitalsadhu) [reply test/index.test.js]: I'll add the metrics object

Comment (digitalsadhu) [reply test/index.test.js]: done

**Pull request merged**